lib/puppet/provider/oneview_hypervisor_cluster_profile/c7000.rb

```diff
@@ -24,6 +24,50 @@
 
   mk_resource_methods
 
+  def exists?
+    create
```

Comment:
exists method should return whether the resource exists or not..? it should be similar to below code
```
def exists?
      @resource_type.find_by(@client, @data).any?
```